code/src/App.js

```diff
@@ -1,9 +1,37 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom"
+
+//API URLS 
+import { ALL_MOVIES_URL } from "./utils/urls"
+
+//Styled components
+import GlobalStyles from "components/styles/Global";
+import AllMovies from 'components/AllMovies'
+import MovieDetails from 'components/MovieDetails'
+
+const App = () => {
+
+  const [allMovies, setAllMovies] = useState([])
+
+  useEffect(() => {
+    fetch(ALL_MOVIES_URL)
+      .then(res => res.json())
+      .then(res => {
+        setAllMovies(res.results)
+      })
+  }, [])
 
-export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <section>
+      <GlobalStyles />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<AllMovies allMovies={allMovies} setAllMovies={setAllMovies}/>} />
```

Comment:
Just wondering why you pass on setAllMovies as a prop. From what I can see you don't use it inside the AllMovies component?